workbench/urls.py

```diff
@@ -8,8 +8,6 @@
 
 admin.autodiscover()
 
-init_scenarios()
```

Comment:
This was how I was getting the scenarios initialized for the application previously.  I'm not sure why I didn't have the call in scenarios.py like I do now.